### PR TITLE
Change made to docs to include example for path_type = vpc

### PR DESCRIPTION
**Pull request by shrsr**

### CODE REVIEW

File: website/docs/r/schema_site_anp_epg_static_port.html.markdown [hunk 1]
```diff
@@ -30,6 +32,46 @@ resource "mso_schema_site_anp_epg_static_port" "static_port" {
...
+  schema_id = "5c4d5bb72700000401f80948"
+  site_id = "5c7c95b25100008f01c1ee3c"
+  template_name = "Template1"
+  anp_name = "ANP"
+  epg_name = "DB"
+  path_type = "dpc"
+  deployment_immediacy = "lazy"
+  pod = "pod-7"
+  leaf = "109"
+  path = "eth1/10"
```
Comment (lhercot) [inline website/docs/r/schema_site_anp_epg_static_port.html.markdown:68]: dpc is also a port group name here not an interface.

File: website/docs/r/schema_site_anp_epg_static_port.html.markdown [hunk 2]
```diff
@@ -30,6 +32,46 @@ resource "mso_schema_site_anp_epg_static_port" "static_port" {
...
+  vlan = 200
+  mode = "untagged"
+
+}
+```
+
 ## Argument Reference ##
```
Comment (lhercot) [inline website/docs/r/schema_site_anp_epg_static_port.html.markdown:76]: Also add an explanation in the Argument reference for leaf. 
Something like:
When `path_type` is `port` or `dpc`, then leaf is a string of the leaf ID. When `path_type` is `vpc`, then `leaf` is a list with both leaf IDs. The `leaf` value is usually something like '101' or '101-102' depending on `path_type`.

**Review by lhercot — CHANGES_REQUESTED** [review]

File: website/docs/r/schema_site_anp_epg_static_port.html.markdown (hunk 2 above)
Comment (shrsr) [reply website/docs/r/schema_site_anp_epg_static_port.html.markdown:76]: okay

File: website/docs/r/schema_site_anp_epg_static_port.html.markdown (hunk 1 above)
Comment (shrsr) [reply website/docs/r/schema_site_anp_epg_static_port.html.markdown]: ok

**Review by lhercot — APPROVED** [review]
LGTM

**Pull request merged**